test/electron-locator.ts

```diff
@@ -1,12 +1,12 @@
-import * as fs from 'fs';
+import * as fs from 'fs-extra';
 import * as path from 'path';
 import { expect } from 'chai';
 import { spawnPromise } from 'spawn-rx';
 
-import { locateElectronPrebuilt } from '../src/electron-locater';
+import { locateElectronModule } from '../src/electron-locator';
 
 function packageCommand(command: string, packageName: string) {
-  return spawnPromise('npm', [command, packageName], {
+  return spawnPromise('npm', [command, '--no-save', packageName], {
```

Comment:
`--no-save` prevents the top-level `package.json` from being overwritten. This is more apparent when developing locally, rather than in CI.